### PR TITLE
Grab more stench jellies where possible

### PR DESCRIPTION
**Pull request by Pantocyclus**

### CODE REVIEW

File: src/combat.ts [hunk 1]
```diff
@@ -286,6 +286,10 @@ export class Macro extends StrictMacro {
           Macro.trySkill($skill`Feel Nostalgic`)
         )
       )
+      .externalIf(
+        myFamiliar() === $familiar`Space Jellyfish`,
+        Macro.if_($monster`garbage tourist`, Macro.trySkill($skill`Extract Jelly`))
```
Comment (horrible-little-slime) [inline src/combat.ts:291]: All tourists in the zone are stench-aligned; I think libram allows monster-arrays now, so you can use $monsters or you can use `monstername * tourist` or something like that

File: src/index.ts [hunk 2]
```diff
@@ -196,6 +196,20 @@ function barfTurn() {
...
+      !embezzlerUp &&
+      have($familiar`Space Jellyfish`) &&
+      get(`_spaceJellyfishDrops`) < 5 &&
+      clamp(
+        myAdventures() - digitizedMonstersRemaining(),
+        globalOptions.saveTurns,
+        myAdventures()
+      ) -
+        globalOptions.saveTurns <
+        20 &&
```
Comment (horrible-little-slime) [inline src/index.ts:209]: I might be about to call the kettle black here, but this math feels really hard to follow--could we get some comments and/or slicing it out into a function?

**Comment by horrible-little-slime** [comment]
Lookin good, just a few remarks to address

File: src/index.ts (hunk 2 above)
Comment (Pantocyclus) [reply src/index.ts:209]: Upon second thoughts the clamp() is not really needed, since I can accept negative values on the LHS just fine. That should simplify things greatly.

File: src/index.ts [hunk 3]
```diff
@@ -200,13 +200,7 @@ function barfTurn() {
...
       have($familiar`Space Jellyfish`) &&
       get(`_spaceJellyfishDrops`) < 5 &&
-      clamp(
-        myAdventures() - digitizedMonstersRemaining(),
-        globalOptions.saveTurns,
-        myAdventures()
-      ) -
-        globalOptions.saveTurns <
-        20 &&
+      myAdventures() - digitizedMonstersRemaining() - globalOptions.saveTurns < 20 &&
```
Comment (horrible-little-slime) [inline src/index.ts:203]: Way, way cleaner looking, 👨‍🍳 💋
